docs/BR.md

```diff
@@ -935,6 +962,8 @@ The CA MAY resend the email, fax, SMS, or postal mail in its entirety, including
 
 The Random Value SHALL remain valid for use in a confirming response for no more than 30 days from its creation. The CPS MAY specify a shorter validity period for Random Values, in which case the CA MUST follow its CPS.
 
+CAs using this method MUST implement Multi-Perspective Issuance Corroboration as specified in [Section 3.2.2.9](#3229-multi-perspective-issuance-corroboration). To count as corroborating, a Network Perspective MUST observe the same IP Address Contact as the Primary Network Perspective.
```

Comment:
I think this triggers MPI on wrong challenge methods for IP addresses?
not sure how one can check RIR documents multiperspectively
think this like is for 3.2.2.5.3  because that's reverse DNS lookup, where multiple perspective make sense.

Reply:
Thank you for bringing this up. I agree that putting multi perspective checking in 3.2.2.5.3 makes sens, and it does not make sense here.

To clarify, a potential fix for this would be to remove this paragraph (line 965) that you commented and put something along these lines at the end of 3.2.2.5.3:

```
CAs using this method MUST implement Multi-Perspective Issuance Corroboration as specified in [Section 3.2.2.9](#3229-multi-perspective-issuance-corroboration). To count as corroborating, a Network Perspective MUST observe the same token as the FQDN as the Primary Network Perspective.
```

Reply:
3.2.2.5 is looking for domain by rDNS and do verification for that domain be used for that IP address, so matching thing MP need to see be domain name there

Reply:
Sorry, I had made a typo in that line recommendation for 3.2.2.5. I meant to delete token and have it read:

```
CAs using this method MUST implement Multi-Perspective Issuance Corroboration as specified in [Section 3.2.2.9](#3229-multi-perspective-issuance-corroboration). To count as corroborating, a Network Perspective MUST observe the same FQDN as the Primary Network Perspective.
```

Reply:
Addressed in https://github.com/ryancdickson/staging/commit/8f35b320fc38cfd16a768d7e18fe73d1fcd92fea